.storybook/main.ts

```diff
@@ -55,6 +70,9 @@ const config: StorybookConfig = {
         });
     },
 
+    core: {
+        disableTelemetry: true,
+    },
```

Comment:
Is this change related to the others?

Reply:
It's not.  I had been comparing to the WB config and it had that in there. I'll remove it and put it in a separate PR.